app/actions/EventActions.ts

```diff
@@ -375,14 +370,15 @@ export function unfollow(followId: EntityId, eventId: EntityId) {
   });
 }
 
-export function isUserFollowing(eventId: EntityId) {
-  return callAPI<boolean>({
-    types: Event.IS_USER_FOLLOWING,
+export function fetchFollowers(eventId: EntityId, currentUserId: EntityId) {
+  return callAPI({
+    types: Event.FETCH_FOLLOWERS,
```

Comment:
Quick comment before the auto-merging hits in for good - I haven't checked what the endpoint returns, but I'm assuming it is just the one object as both the event and user is specified. If that is correct, could you rename this in a way that reflects it's singular nature? If I'm wrong the plural you're currently using is just fine and I have no further comments(:

Reply:
I think it would be good to add an endpoint for checking just one user, though. It is a bit unnecessary to fetch all of them

Reply:
Lol aight that's weird.. then I feel like it makes no sense to have to supply the current user (which is weird in either way), but I guess the backend endpoint requires it for some reason 🤷

Reply:
No, actually we supply the current user for frontend reasons. In order to check if the current user is following the event, we need to know who they are. We don't have access to that in the `events` reducer, so we need to attach it to the `fetchFollowers`-action

Reply:
This code is kind of a mess, but I think fixing it should be done in a separate PR

Reply:
> I think it would be good to add an endpoint for checking just one user, though. It is a bit unnecessary to fetch all of them

absolutely - as far as I know there is nowhere in the webapp where we allow other users to see who else is following an event - so it should be fine to alter the current implementation I think

Reply:
Yeah, no. Almost all the followers stuff is unused, I think a lot more functionality around it was planned at some point.